website/docs/concurrency_control.md

```diff
@@ -61,6 +68,18 @@ With multiple writers using OCC, these are the write guarantees to expect:
 - *INCREMENTAL PULL Guarantee*: Data consumption and checkpoints are NEVER out of order. If there are inflight commits 
   (due to multi-writing), incremental queries will not expose the completed commits following the inflight commits. 
 
+#### Multi Writer Guarantees with Non-Blocking Concurrency Control Mode
```

Comment:
Should we mark this experimental as well int eh heading ?

Reply:
good point.. i'll mark it in the heading.